### PR TITLE
PUBDEV-5072: merge duplicate.  

### PR DESCRIPTION
**Pull request by wendycwong**
Need to use the merge method radix:

h2o.merge(h2o.asfactor(test3), h2o.asfactor(test4), by = "NAME", all.x = TRUE, method="radix")

For details checkout the runit test.

### CODE REVIEW

File: h2o-py/tests/pyunit_utils/utilsPY.py [hunk 1]
```diff
@@ -2973,7 +2973,7 @@ def cannaryHDFSTest(hdfs_name_node, file_name):
             return False
 
 
-def extract_scoring_history_field(aModel, fieldOfInterest):
+def extract_scoring_history_field(aModel, fieldOfInterest, grab1Ele=False):
```
Comment (michalkurka) [inline h2o-py/tests/pyunit_utils/utilsPY.py]: Please rename to "grabFirst" or "takeFirst". This one confused me a lot :)

File: h2o-py/tests/testdir_algos/gbm/pyunit_PUBDEV_4090_grid_model_seeds.py [hunk 2]
```diff
@@ -81,8 +81,8 @@ def random_grid_model_seeds_PUBDEV_4090():
                                                            "got %s" % (expectedSeeds, model2Seeds)
 
     # compare training_rmse from scoring history
-    metric_list1 = pyunit_utils.extract_scoring_history_field(air_grid1.models[0], "training_rmse")
-    metric_list2 = pyunit_utils.extract_scoring_history_field(air_grid2.models[0], "training_rmse")
+    metric_list1 = pyunit_utils.extract_scoring_history_field(air_grid1.models[0], "training_rmse", True)
```
Comment (michalkurka) [inline h2o-py/tests/testdir_algos/gbm/pyunit_PUBDEV_4090_grid_model_seeds.py]: Does this change have anything to do with the subject of the PR "PUBDEV-5072: merge duplicate"?

File: h2o-r/tests/testdir_munging/runit_pubdev_5072_merge_duplicate.R [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+source("../../scripts/h2o-r-test-setup.R")
+##
+# Test out the merge() functionality
+##
+
+test.merge <- function() {
+  test3 <- h2o.importFile(locate("smalldata/jira/test3.csv"), header=TRUE)
+  test4 <- h2o.importFile(locate("smalldata/jira/test4.csv"), header=TRUE)
+  ans <- as.factor(h2o.importFile(locate("smalldata/jira/pubdev_5072_answer.csv"), header=TRUE))
+  browser()
```
Comment (michalkurka) [inline h2o-r/tests/testdir_munging/runit_pubdev_5072_merge_duplicate.R]: Please remove

**Comment by michalkurka** [comment]
@wendycwong I don't see any change in the code that would fix PUBDEV-5072. What am I missing?

File: h2o-py/tests/pyunit_utils/utilsPY.py (hunk 1 above)
Comment (wendycwong) [reply h2o-py/tests/pyunit_utils/utilsPY.py]: Sure

File: h2o-r/tests/testdir_munging/runit_pubdev_5072_merge_duplicate.R (hunk 3 above)
Comment (wendycwong) [reply h2o-r/tests/testdir_munging/runit_pubdev_5072_merge_duplicate.R]: done

File: h2o-py/tests/testdir_algos/gbm/pyunit_PUBDEV_4090_grid_model_seeds.py (hunk 2 above)
Comment (wendycwong) [reply h2o-py/tests/testdir_algos/gbm/pyunit_PUBDEV_4090_grid_model_seeds.py]: Nope

Comment (wendycwong) [reply h2o-py/tests/testdir_algos/gbm/pyunit_PUBDEV_4090_grid_model_seeds.py]: It fixes an intermittent.  I can do a separate PR for it if you want.

File: h2o-r/tests/testdir_munging/runit_pubdev_5072_merge_duplicate.R (hunk 3 above)
Comment (wendycwong) [reply h2o-r/tests/testdir_munging/runit_pubdev_5072_merge_duplicate.R]: Done

**Pull request merged**